src/components/LoanCards/Buttons/LendButton.vue

```diff
@@ -70,7 +70,7 @@ export default {
 								`Failed: ${error.message.substring(0, 40)}...`
 							);
 							Sentry.captureMessage(`Add to Basket: ${error.message}`);
-							if (error?.extensions?.code === 'shop.invalidBasketId') {
+							if (['shop.invalidBasketId', 'shop.basketRequired'].includes(error?.extensions?.code)) {
```

Comment:
This might be a good line of code to extract to a util, since we have the same 2 error strings in 3 places now.

Reply:
After a few new attempts, shop was set to `null` because of hot reload (I think this is an edge case only seen in local development) but do you think I should set the new basket anyway? If not I can just revert these files

Reply:
Oh, that's a good catch. I think a util to encasulate the is repeated check logic is enough for now. THanks!